### PR TITLE
kubeadm v1.15 blog post

### PR DESCRIPTION
**Pull request by luxas**
cc @kubernetes/sig-cluster-lifecycle @castrojo 

FYI, I have checked "Allow edits from maintainers". Would appreciate grammar corrections, formatting fixes etc.
Thanks!

### CODE REVIEW

File: content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+# Automated High Availability in kubeadm v1.15: Batteries Included But Swappable
+
+[kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/) is a tool that enables Kubernetes administrators
+to quickly and easily bootstrap minimum viable clusters that are fully compliant with 
+[Certified Kubernetes](https://github.com/cncf/k8s-conformance/blob/master/terms-conditions/Certified_Kubernetes_Terms.md) guidelines.
+It’s been under active development by [SIG Cluster Lifecycle](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle)
+since 2016 and we graduated it from beta to
```
Comment (DanyC97) [inline content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md]: maybe we can rephrase it to `and graduated from beta to`

File: content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+[stable and generally available (GA) in end of 2018](https://kubernetes.io/blog/2018/12/04 production-ready-kubernetes-cluster-creation-with-kubeadm/).
```
Comment (DanyC97) [inline content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md]: maybe `generally available (GA) at the end of 2018`

File: content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+## The scope of kubeadm
+
+kubeadm is focused on performing the actions necessary to get a minimum viable, secure cluster up and running in a
+user-friendly way. kubeadm's scope is limited to the local machine’s filesystem and the Kubernetes API, and it is
+intended to be a _composable building block for higher-level tools_. 
+
+The core of the kubeadm interface is quite simple: new control plane nodes are created by running 
+<strong><code>kubeadm init</code></strong>, worker nodes are joined to the control plane by running
+<strong><code>kubeadm join</code></strong>. Also included are common utilities for managing already bootstrapped
+clusters, such as control plane upgrades and token and certificate renewal.
```
Comment (DanyC97) [inline content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md]: maybe `control plane upgrades, token and certificate renewal` ?

File: content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+To keep kubeadm lean, focused, and vendor/infrastructure agnostic, the following tasks are out of scope:
+
+*   Infrastructure provisioning
+*   Third-party networking
+*   Non-critical add-ons for e.g. monitoring, logging, and visualization
```
Comment (DanyC97) [inline content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md]: maybe remove `for`

File: content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+
+In kubeadm v1.15, we have improved the structure and are releasing the new **v1beta2** format. Important to note that the existing **v1beta1** format released in v1.13 will still continue to work for several releases. This means you can upgrade kubeadm to v1.15, and still use your existing v1beta1 configuration files. When you’re ready to take advantage of the improvements made in v1beta2, you can perform an automatic schema migration using the `kubeadm config migrate` command. Important to note in this release is that support for the _v1alpha3_ version has been dropped.
+
+During the course of the year, we’re looking forward to graduate the schema to General Availability `v1`.` If you’re interested in this effort, you can also join [WG Component Standard](https://github.com/kubernetes/community/tree/master/wg-component-standard).
+
+## What’s next?
+
+### 2019 plans
+
+We are focusing our efforts this year around graduating the configuration file format to GA (`kubeadm.k8s.io/v1`)`, graduating this super-easy High Availability flow to stable, and providing better tools around rotating certificates needed for running the cluster automatically.
```
Comment (DanyC97) [inline content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md]: nit: `this year` can be taken out since the header is 2019 plans

File: content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+
+You should expect also more work around certificate management in kubeadm in the next releases, with the introduction of ECDSA keys and with improved support for CA key rotation. Additionally, the commands staged under `kubeadm alpha` are expected to move top-level soon.
+
+
+### Improved Configuration File Format
+
+You can argue that there are hardly two Kubernetes clusters that are configured equally, and hence there is a need to customize how the cluster is set up depending on the environment. One way of configuring a component is via flags. However, this has some scalability limitations:
+
+
+* **Hard to maintain.** When $component’s flag set grows over 30+ flags, configuring it becomes really painful
```
Comment (sftim) [inline content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md]:
```suggestion
* **Hard to maintain.** When a component’s flag set grows over 30+ flags, configuring it becomes really painful.
```

File: content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+
+[SIG Cluster Lifecycle](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle) has many different
+subprojects, where kubeadm is one of them. In the following picture you can see that there are many pieces in the
+puzzle, and we have a lot still to do.
+
+![SIG Cluster Lifecycle Projects](/static/images/blog/2019-06-24-kubeadm-ha-v115/projects.png)
+
+Some handy links if you want to start contribute:
+
+*   We have a recorded [New Contributor Onboarding](https://www.youtube.com/watch?v=Bof9aveB3rA) session on YouTube.
```
Comment (sftim) [inline content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md]:
```suggestion
*   You can watch the SIG Cluster Lifecycle [New Contributor Onboarding](https://www.youtube.com/watch?v=Bof9aveB3rA) session on YouTube.
```

File: content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+*   Look out for “good first issue”, “help wanted” and “sig/cluster-lifecycle” labeled issues in our repositories 
+    (e.g. [kubernetes/kubeadm](https://github.com/kubernetes/kubeadm))
+*   Join **#sig-cluster-lifecycle**, **#kubeadm**, **#cluster-api**, **#minikube**, **#kind**, etc. in Slack
```
Comment (sftim) [inline content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md]: Which Slack organisation? (I assume it's the main Kubernetes Slack).
Is it worth linking there?

File: content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md [hunk 9]
```diff
@@ -0,0 +1,202 @@
+# Automated High Availability in kubeadm v1.15: Batteries Included But Swappable
```
Comment (sftim) [inline content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md]: This blog post needs usual front matter. Have a look at other blog posts for an example.

File: content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md [hunk 10]
```diff
@@ -0,0 +1,202 @@
...
+*   Check out the [SIG Cluster Lifecycle Intro](https://youtu.be/bA2M41J4wvg) or the
+    [kubeadm Deep Dive](https://youtu.be/spXSSIbZTqM) sessions from KubeCon Barcelona
+
+
+### Thank You
+
+This release wouldn’t have been possible without the help of the great people that have been contributing to SIG Cluster Lifecycle and kubeadm. We would like to thank all the kubeadm contributors and companies making it possible for their developers to work on Kubernetes!
+
+
+## Written by:
```
Comment (sftim) [inline content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md]: Usually, Kubernetes blog posts list the author(s) just below the front matter.

**Comment by castrojo** [comment]
Ok I've addressed all the review comments, updated the logo, and added the front matter, PTAL.

**Comment by luxas** [comment]
I also updated some minor things on it now (e.g. addressed the author and subproject owners attributions)
This is ready to merge IMO

@timothysc want to sign off?

**Comment by stealthybox** [comment]
> During the course of the year, we’re looking forward to graduate the schema to General Availability v1.` If you’re interested in this effort, you can also join WG Component Standard.

Nice :ok_hand: 

/lgtm

**Comment by luxas** [comment]
Just merge whenever you're ready. Thanks!
